homeassistant/components/risco/binary_sensor.py

```diff
@@ -35,6 +35,10 @@ async def async_setup_entry(
             RiscoLocalAlarmedBinarySensor(local_data.system.id, zone_id, zone)
             for zone_id, zone in local_data.system.zones.items()
         )
+        async_add_entities(
```

Comment:
Please collect the entities in a list and make a single call to `async_add_entities`.

Reply:
OK, here: https://github.com/home-assistant/core/pull/82043